assets/js/app.js

```diff
@@ -224,6 +212,18 @@ function enableTitleInput(grid, input) {
     input.setSelectionRange(0, 99999);
 }
 
+function updateWidget(widget) {
+    const url = Routing.generate('update_widget', {
+        id: widget.getAttribute('gs-id'),
+        x: widget.getAttribute('gs-x'),
+        y: widget.getAttribute('gs-y'),
+        width: widget.getAttribute('gs-w'),
+        height: widget.getAttribute('gs-h'),
```

Comment:
why do you use getAttribute instead of using property access ?

Reply:
The `resize` and the `drag` events return the DOM element as second parameter, not the JS object. With this, properties doesn't exist so I must use `getAttribute()` function.

I must use those events because the `change` event reacts when the user resizes his browser.